### ISSUE TITLE
Is there a method for  automatic adjustment of learning rate?

### ISSUE THREAD

**Issue by davidduo** [apache/incubator-mxnet#6652]
with callback function, we can make learning rate drop by some rate after fixed epochs or batches in training. Is there a method for automatic adjustment of learning rate, for example according some evaluation metric in training? For example, if the accuracy of validation set does not increase for 3 epoch, then learning rate drop by 1/10?

**Comment by saicoco** [comment]
params for lr can be set in optimizer, refet [this](http://mxnet.io/api/python/optimization.html#the-mxnet-lr-scheduler-package), `mx.optimizer.optimizer(lr_scheduler=lrs)` can achieve you task

**Comment by yajiedesign** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
